test/Microsoft.DotNet.ProjectJsonMigration.Tests/MSBuild.exe.config

```diff
@@ -0,0 +1 @@
+hey
```

Comment:
Yes this is required to get the Microsoft.Build.Construction apis to work.

Having the text file was the simplest solution here.
@rainersigwald filed a bug for this scenario:
https://github.com/Microsoft/msbuild/issues/927

I'm open to having it say something besides "hey" though. 😄